### DISCUSSION TITLE
sonoff ZBmini firmware 2.0.0 work fine but last Seen problem

### DISCUSSION THREAD

**Discussion by steve41200** [Koenkk/zigbee2mqtt#12486] [Devices]
Hi.
I have 6 ZBMINI sonoff 3 on firmware 2.0.0 ( date: 20210320)who have the same problem.

They works fine when I trigger them into HomeAssistant or Z2M. It update the last seen parameter, but last seen never actualise alone like other rooter.
I have 3 other ZBMINI with no number firmware information but date firmware older (20200927). The last seen parameter update alone like other rooter.
Is their is a trick to change firmware ?
Without that problem they works fine.
I use the last seen for tell me each day if a device is offline. So they tell me this 3 are offline if they don't were trigger one day ...

I have try to plug on just near the coordinator but I have the same effect. No update last seen if no trigger

I have the latest version of add on z2m Current version: 1.25.1-1
the coordinator is Texas Instruments LAUNCHXL-CC1352P-2

<img width="1440" alt="Capture d’écran 2022-05-15 à 13 21 51" src="https://user-images.githubusercontent.com/60097951/168470196-87280b36-77bb-4336-b790-9ff613b23c80.png">

<img width="1440" alt="Capture d’écran 2022-05-15 à 13 21 51" src="https://user-images.githubusercontent.com/60097951/168470372-88463114-3400-4b59-a569-c735f9225217.png">

**Comment by tzeryin1981** [comment]
I have the same issue, one of it will offline , I have to trigger the device on off to wake the device up.
All my zbmini is v2.0 20210320

**Comment by steve41200** [comment]
Hum ill happe TO notre be Alone.
You have thé availability activated i suppose. Try to disable it.
But the best Will be an update… but it don’t Come

**Comment by distante** [comment]
Looks like 20210320 is faulty? I have 3 with the 2020 version and they work as expected. The 20210320 will stop responding after a while.
